pkg/restapi/operation/operations_test.go

```diff
@@ -524,7 +543,7 @@ func TestCreateDocument(t *testing.T) {
 		require.NoError(t, err)
 
 		urlVars := make(map[string]string)
-		urlVars[vaultIDPathVariable] = testVaultID
+		urlVars[vaultIDPathVariable] = testReferenceID
```

Comment:
Similar comment as before - should really be some sort of testVaultID

Reply:
fixed